static/app.template.html

```diff
@@ -67,15 +67,28 @@
 <body>
   <h1>Welcome {{.User}}!</h1>
 
-  <div class="pure-g charts-container">
-    <div class="pure-u-7-8 pure-u-md-2-5 chart-container">
-      <div id="summChart"></div>
+  <div class="pure-g">
+    <div class="pure-u-1 pure-u-md-1-3">
+      <div class="operation-container">
+      <form action="listops" method="post" class="pure-form pure-form-aligned">
```

Comment:
We don't need an `action` nor `method` here, right?

Same for the other <form> below.